DER/DerivativesContracts/Options.rdf

```diff
@@ -378,54 +336,37 @@
 		<rdfs:label xml:lang="en">out-of-the-money</rdfs:label>
 		<skos:definition xml:lang="en">moneyness classifier that refers to an option whose value in terms of its strike price is not favorable in comparison to the prevailing market price of the underlying asset</skos:definition>
 		<fibo-fnd-utl-av:abbreviation xml:lang="en">OTM</fibo-fnd-utl-av:abbreviation>
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">An OTM call option will have a strike price that is higher than the market price of the underlying asset. Alternatively, an OTM put option has a strike price that is lower than the market price of the underlying asset.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:NamedIndividual>
 	
-	<owl:Class rdf:about="&fibo-der-der-opt;PutOption">
+	<owl:Class rdf:about="&fibo-der-drc-opt;PutOption">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;Option"/>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-pas-pas;hasBuyer"/>
-				<owl:someValuesFrom>
-					<owl:Class>
-						<owl:unionOf rdf:parseType="Collection">
-							<rdf:Description rdf:about="&fibo-der-der-opt;OptionHolder">
-							</rdf:Description>
-							<rdf:Description rdf:about="&fibo-der-drc-bsc;ReceivingParty">
-							</rdf:Description>
-						</owl:unionOf>
-					</owl:Class>
-				</owl:someValuesFrom>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-pas-pas;hasSeller"/>
-				<owl:someValuesFrom>
-					<owl:Class>
-						<owl:unionOf rdf:parseType="Collection">
-							<rdf:Description rdf:about="&fibo-der-der-opt;OptionIssuer">
-							</rdf:Description>
-							<rdf:Description rdf:about="&fibo-der-drc-bsc;PayingParty">
-							</rdf:Description>
-						</owl:unionOf>
-					</owl:Class>
-				</owl:someValuesFrom>
-			</owl:Restriction>
-		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">put option</rdfs:label>
 		<skos:definition xml:lang="en">option giving the buyer (holder) the right, but not the obligation, to to sell the assets specified at a fixed price or formula, on or before a specified date</skos:definition>
 		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, 2019.</fibo-fnd-utl-av:adaptedFrom>
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The seller of the put option assumes the obligation of buying the assets specified should the buyer exercise the option.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-der-opt;SingleStockOption">
-		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;EquityOption"/>
+	<owl:Class rdf:about="&fibo-der-drc-opt;SingleStockOption">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-opt;EquityOption"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
```

Comment:
not sure why "hasIdentity" here: why not hasUnderlier exactlyOne ListedShare? . What class do you expect the underlier to be?

Reply:
@rivettp See comment above - I deleted the class so this is moot.